packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterForm.tsx

```diff
@@ -73,20 +74,20 @@ function GridFilterForm(props: GridFilterFormProps) {
   const ownerState = { classes: rootProps.classes };
   const classes = useUtilityClasses(ownerState);
 
-  const getCurrentColumn = React.useCallback(() => {
+  const currentColumn = React.useMemo<GridStateColDef | null>(() => {
     if (!item.columnField) {
       return null;
     }
     return apiRef.current.getColumn(item.columnField)!;
   }, [apiRef, item]);
 
-  const getCurrentOperator = React.useCallback(() => {
-    const currentColumn = getCurrentColumn();
```

Comment:
Am I missing something ... `currentColumn` is now a function right? How is it being called to get the target column `ColDef`?